### PR TITLE
FEATURE: Admin Banner when Logster logs is getting flooded.

### PR DESCRIPTION
**Pull request by tgxworld**
https://meta.discourse.org/t/admin-banner-notification-of-error-floods-at-logs/39379

### CODE REVIEW

File: app/assets/javascripts/discourse/templates/application.hbs [hunk 1]
```diff
@@ -6,6 +6,7 @@
       {{custom-html "top"}}
     {{/if}}
     {{global-notice}}
```
Comment (SamSaffron) [inline app/assets/javascripts/discourse/templates/application.hbs]: Can't this be simply added into global-notice, uneasy adding another component for everyone, its more work for ember to do on every page.

File: config/site_settings.yml [hunk 2]
```diff
@@ -764,6 +764,14 @@ rate_limits:
   tl2_additional_likes_per_day_multiplier: 1.5
   tl3_additional_likes_per_day_multiplier: 2
   tl4_additional_likes_per_day_multiplier: 3
+  alert_admins_if_errors_per_minute:
+    client: true
+    shadowed_by_global: true
+    default: 100
```
Comment (SamSaffron) [inline config/site_settings.yml]: what value is used to disable altogether? that should be default.

Comment (SamSaffron) [inline config/site_settings.yml]: also we got to think about how we are going to get this all going in our environment, we may need a custom plugin for meta to poll rate, cause error reports are reported from multiple sources.

File: app/assets/javascripts/discourse/templates/application.hbs (hunk 1 above)
Comment (tgxworld) [inline app/assets/javascripts/discourse/templates/application.hbs]: `global-notice` deals with static content that is displayed site wide for all users. 
`logs-notice` would need to keep track of a few properties to determine if it should be displayed. I don't think we should mix it into `global-notice`.

Comment (tgxworld) [inline app/assets/javascripts/discourse/templates/application.hbs]: O wait .. let me try moving the logic into a model

File: config/site_settings.yml [hunk 3]
```diff
@@ -761,6 +761,14 @@ rate_limits:
   tl2_additional_likes_per_day_multiplier: 1.5
   tl3_additional_likes_per_day_multiplier: 2
   tl4_additional_likes_per_day_multiplier: 3
+  alert_admins_if_errors_per_minute:
+    client: true
```
Comment (SamSaffron) [inline config/site_settings.yml]: default should be false for now till we tune this

File: config/locales/server.en.yml [hunk 4]
```diff
@@ -977,6 +977,9 @@ en:
     max_invites_per_day: "Maximum number of invites a user can send per day."
     max_topic_invitations_per_day: "Maximum number of topic invitations a user can send per day."
 
+    alert_admins_if_errors_per_minute: "Number of errors per minute in order to trigger an admin alert. NOTE: requires restart."
+    alert_admins_if_errors_per_hour: "Number of errors per hour in order to trigger an admin alert. NOTE: requires restart."
```
Comment (SamSaffron) [inline config/locales/server.en.yml]: description should be fixed, to note that 0 means it is disabled

**Comment by SamSaffron** [comment]
Looking good, go ahead and merge in first thing next week

**Pull request merged**